### PR TITLE
Add Eclipse page in the tools.

### PR DESCRIPTION
**Pull request by akurtakov**
List Eclipse in the tools page.

### CODE REVIEW

File: tools/eclipse/about.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+# What is Eclipse
+
+[Eclipse](https://www.eclipse.org/ide/) is a full featured, multi language IDE aiding developers in every step of software development lifecycle. Starting new project, working on established project with complex structure, monitoring and performance optimizing application, writing documentation, interacting with issue tracking systems or CI all from inside Eclipse IDE.
+
+## Installing Eclipse on Fedora
+
+If you are running Fedora Workstation it is strongly recommended to install Eclipse as shipped with Fedora as most of the times it contains important fixes, backported from upstream and not yet in released version:
+
+    $ dnf install eclipse
```
Comment (phracek) [inline tools/eclipse/about.md]: First of all command has to be sudo dnf install
And the whole command has to be closed in 3xbackticks.

LIke 
```
sudo dnf install eclipse
```

File: tools/eclipse/about.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+description: Full featured development environment for various programming languages integrating with plethora of different tools to make development process as integrated as possible
+---
+
+# What is Eclipse
+
+[Eclipse](https://www.eclipse.org/ide/) is a full featured, multi language IDE aiding developers in every step of software development lifecycle. Starting new project, working on established project with complex structure, monitoring and performance optimizing application, writing documentation, interacting with issue tracking systems or CI all from inside Eclipse IDE.
+
+## Installing Eclipse on Fedora
+
+If you are running Fedora Workstation it is strongly recommended to install Eclipse as shipped with Fedora as most of the times it contains important fixes, backported from upstream and not yet in released version:
```
Comment (phracek) [inline tools/eclipse/about.md]: I would like to rephrase it like, I don't think, it is strongly recommended.
It depends on user.

What about something like that:
On Fedora install the eclipse package:

File: tools/eclipse/about.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+* Systemtap - eclipse-systemtap
+* Kernel Perf - eclipse-perf
+* Oprofile - eclipse-oprofile
+* Bug trackers and CIs - eclipse-mylyn-* packages
+
+The list above doesn't contain all Eclipse plugins available in Fedora but is supposed to serve as an example of the range of different technologies integrated.
+
+Alternative method for installing Eclipse on Fedora is by using `gnome-software`. Some of the packaged plugins ship proper metadata so they can be seen and installed directly from it.
+
+Last but not least there is [Eclipse Marketplace](http://marketplace.eclipse.org/) (`dnf install eclipse-mpc`) giving you access to even bigger list of plugins to integrate even more tools in your ide. Once Eclipse is running go to `Help\Eclipse Marketplace...` and search for plugins serving your needs.
```
Comment (phracek) [inline tools/eclipse/about.md]: Again sudo dnf install ...

**Comment by phracek** [comment]
Please, send us updated PR. Thanks for your PR and supporting Fedora Developer Portal.

Thanks again

**Comment by akurtakov** [comment]
Updated PR available.

**Pull request merged**